lib/Service/OpenAiSettingsService.php

```diff
@@ -180,7 +187,7 @@ public function getQuotas(): array {
 	 * @return boolean
 	 */
 	public function getChatEndpointEnabled(): bool {
-		return $this->config->getAppValue(Application::APP_ID, 'chat_endpoint_enabled', '0') === '1';
+		return $this->appConfig->getValueString(Application::APP_ID, 'chat_endpoint_enabled', '0') === '1';
```

Comment:
let's default to chat endpoint since many services assume so and it is more usable in general? What do you say?

Reply:
sure